### DISCUSSION TITLE
Please add Bhutan

### DISCUSSION THREAD

**Discussion by KinWang-2013** [community/community#27278] [Other Feature Feedback, Questions, & Ideas]
Please add Bhutan in the country list

**Comment by jfullstackdev** [comment]
of what ? GitHub Sponsors ? I think so, 

![image](https://user-images.githubusercontent.com/47092464/181254645-3787848b-f059-4a1b-ba64-e13d13ff5ee1.png)

did you join the waitlist so GitHub will know there is a request at that country ? 

I also requested the Philippines to be added

**Reply by KinWang-2013** [reply]
yes, sponsor and also while adding phone number Bhutan is not in the list

**Reply by jfullstackdev** [reply]
hopefully there are other requests from Bhutan, I think it's by priority when there are so many requests from a certain country
